### PR TITLE
A small review of the guidelines

### PR DESCRIPTION
**Pull request by fasterius**
As requested, a small review of the NBIS developer guidelines. Most are minor additions of links to other resources or additions/clarifications to already existing content, coming from a NBIS Support staff perspective. Maybe this is not what you want to have in these guidelines, so just reject as you see fit. Commits bodies describe my reasons for adding that particular commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,8 @@ does should ideally already be evident from the code itself. If the code
 is cryptic and can't easily be simplified, explanations might well be
 needed.
 
-A good comment clarifies intent.
+A good comment clarifies intent; you can read more about this
+[here](git@github.com:fasterius/development-guidelines.git).
```
Comment (jhagberg) [inline README.md:96]: You link to your fork of this repo?

File: README.md [hunk 2]
```diff
@@ -301,7 +301,12 @@ members of NBISweden, *but only if there are very special circumstances
 that require this*. The ELIXIR Open Source Principles say "Start a
 project in the open from the very first day, in a publicly accessible,
 version controlled repository [...] The longer a project is run in a
-closed manner, the harder it is to open source it later".
+closed manner, the harder it is to open source it later". If the
```
Comment (jhagberg) [inline README.md:304]: Yes this is a good addition!

**Review by jhagberg — CHANGES_REQUESTED** [review]
Good addition and many thanks for your time! Clarify the link for comment I did not follow it. But I might miss something since I am on the mobile github interface.

File: README.md (hunk 1 above)
Comment (fasterius) [reply README.md]: Oops, that must have been a copy-paste gone wrong! Fixed now.

Comment (fasterius) [reply README.md]: Adding and pushing an additional commit (fixing the link) apparently didn't automatically ask for an additional review, so I did it manually instead.

**Review by jhagberg — APPROVED** [review]
LGTM

**Pull request merged**